src/app/offer/offer.component.html

```diff
@@ -91,23 +91,53 @@ <h1 class="text-4xl font-light tracking-wider leading-relaxed">{{ offer.name }}<
-                    stroke="#3730A3" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
-                </svg>
-                <a href="{{ newsItem.link }}" class="ml-3 text-sm">{{ newsItem }}</a>
+                <a class="ml-3 text-sm" href="{{ document.location | toUrlIPFS }}"
+                   target="_blank" rel="noopener noreferrer">
+                  {{ document.name }}
+                </a>
               </div>
+              <ng-container *ngIf="links$ | async as linksRes">
+                <div class="mt-10 text-lg uppercase font-light">Project News</div>
+                <div class="bg-gray-100 p-4 rounded-lg">
+                  <ng-container *ngIf="linksRes.value as links">
+                    <div *ngFor="let link of links" class="mt-4 flex flex-col">
+                      <div class="flex items-centerr">
+                        <svg width="26" height="26" viewBox="0 0 26 26" fill="none" xmlns="http://www.w3.org/2000/svg">
+                          <path
+                            d="M14.3333 18.3333H13V13H11.6667M13 7.66667H13.0133M25 13C25 14.5759 24.6896 16.1363 24.0866 17.5922C23.4835 19.0481 22.5996 20.371 21.4853 21.4853C20.371 22.5996 19.0481 23.4835 17.5922 24.0866C16.1363 24.6896 14.5759 25 13 25C11.4241 25 9.86371 24.6896 8.4078 24.0866C6.95189 23.4835 5.62902 22.5996 4.51472 21.4853C3.40042 20.371 2.5165 19.0481 1.91345 17.5922C1.31039 16.1363 1 14.5759 1 13C1 9.8174 2.26428 6.76516 4.51472 4.51472C6.76516 2.26428 9.8174 1 13 1C16.1826 1 19.2348 2.26428 21.4853 4.51472C23.7357 6.76516 25 9.8174 25 13Z"
+                            stroke="#3730A3" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                        </svg>
+                        <a href="{{ link.url }}" class="ml-3 text-sm" target="_blank" rel="noopener noreferrer">
+                          {{ link.open_graph?.title ?? link.url }}
+                        </a>
+                      </div>
+                      <div *ngIf="link.open_graph?.image?.url" class="flex justify-center p-2">
+                        <img class="max-h-60"
+                             src="{{ link.open_graph!.image!.url }}"
+                             alt="News item photo" srcset="">
+                      </div>
+                      <div *ngIf="link.open_graph?.description" class="flex p-2 text-sm">
+                        {{ link.open_graph!.description }}
+                      </div>
+                    </div>
+                  </ng-container>
+
+                  <ng-container *ngIf="linksRes.loading">
+                    <app-spinner type="inline-xs"></app-spinner>
+                  </ng-container>
+
+                  <ng-container *ngIf="linksRes.error">
+                    Cannot fetch news links.
+                    <div hidden>{{ linksRes.error | json }}</div>
+                  </ng-container>
```

Comment:
Can you reorder those two containers to be consistent with the order of the rest of the app? (value, error, loading)